lib/screens/department_screen/widgets/about_section.dart

```diff
@@ -0,0 +1,217 @@
+import 'package:expandable/expandable.dart';
+import 'package:flutter/material.dart';
+import 'package:tsec_app/models/department_model/about_model.dart';
+
+import 'package:tsec_app/utils/themes.dart';
+
+class AboutSection extends StatefulWidget {
+  const AboutSection({Key? key}) : super(key: key);
+
+  @override
+  _AboutSectionState createState() => _AboutSectionState();
+}
+
+class _AboutSectionState extends State<AboutSection> {
+  @override
+  Widget build(BuildContext context) {
+    AboutModel aboutModel = AboutModel();
+    Size size = MediaQuery.of(context).size;
+    return Expanded(
+      child: SingleChildScrollView(
+        child: Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 20, vertical: 10),
+          child: SizedBox(
+            width: size.width,
+            height: null,
```

Comment:
Why pass null?

Reply:
because I want the `SizedBox` to be unbounded? does not specifying the height mean the same?

Reply:
Yes not specifying means same